dotnetcore3.1/cookiecutter-aws-sam-quick-start-sqs-dotnet/{{cookiecutter.project_name}}/events/event-sqs.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Event file seems different then the one in Nodejs init template. https://github.com/aws/aws-sam-cli-app-templates/blob/master/nodejs14.x/cookiecutter-quick-start-sqs/%7B%7Bcookiecutter.project_name%7D%7D/events/event-sqs.json

Reply:
@mndeveci this is the correct one that SQS send, the node one content only one message, but SQS should send record:[], my .net code is parsing the message using a class provided by AWSSDK .NET which is the same message if you generate sam local 
```bash
sam local generate-event sqs receive-message
```